client/benchmark/build.gradle

```diff
@@ -17,8 +17,7 @@ tasks.named("assemble").configure {  enabled = true }
 archivesBaseName = 'client-benchmarks'
 mainClassName = 'org.elasticsearch.client.benchmark.BenchmarkMain'
 
-// never try to invoke tests on the benchmark project - there aren't any
-tasks.named("test").configure {enabled = false }
+tasks.named("uploadSnykDependencyGraph").configure {enabled = false }
```

Comment:
this plugin applies `elasticsearch.build` but I don't consider it production code